cmd/create/cluster/cmd.go

```diff
@@ -1879,6 +1863,7 @@ func run(cmd *cobra.Command, _ []string) {
 			subnetID := awssdk.StringValue(subnet.SubnetId)
 			availabilityZone := awssdk.StringValue(subnet.AvailabilityZone)
 			mapSubnetToAZ[subnetID] = availabilityZone
+			mapSubnetToOwnerID[subnetID] = awssdk.StringValue(subnet.OwnerId)
```

Comment:
Can we create a map of subnet ID to some middle structure so we can keep a single map to all information required? Basically unify the AZ and OwnerID

Reply:
Changed, thanks.